apps/judicial-system/backend/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -95,6 +95,11 @@ export class Defendant extends Model {
   @ApiPropertyOptional({ type: String })
   nationalId?: string
 
+  // ATTENTION: This will contain they DOB from **LOKE** but we can migrate later internal DOB currently stored as nationalId in the same schema
```

Comment:
```suggestion
  // ATTENTION: This will contain the DOB from **LOKE** but we can migrate later internal DOB currently stored as nationalId in the same schema
```